swagger-codegen/2.2.0.wso2v3/pom.xml

```diff
@@ -0,0 +1,132 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ ~ Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Fixed in https://github.com/wso2/orbit/pull/265/commits/84f79dc486c94adbc41108d6c7300d8553a3f746